### ISSUE TITLE
[Bug or Feature request]When the keyboard is active in the system menu, the mouse should not take priority

### ISSUE THREAD

**Issue by Loufute** [linuxmint/cinnamon#10590]
**TLDR :** 
When using the keyboard to navigate through the system menu, mouse cursor position should not take priority if not moved.

--

Hi,

When I navigate through the system menu using the keyboard, and the mouse is over menu items, it takes priority and selects the item underneath, such that, if I go from one category to another, or if I use the menu as a launcher, it does not select the item that I would expect.

**Steps to reproduce**
Place cursor at the level of the second line of an application in the system menu.
Open system menu with keyboard (by default: "Super"), start typing an app name or navigating through categories, see the second line being highlighted.

**Expected behavior**
The first line (first app appearing) is highlighted when opening the system menu with the keyboard, and only if the cursor is moved, its position is taken into account to highlight the corresponding menu item.

**Additional info**
- This works as expected regarding the left system menu bar. If you place the cursor on, let's say, Software manager, and that you start typing (e.g., "Fire"), the Software manager highlight gets disabled, and "Firefox" (or any app appearing at top when typing "Fire") is highlighted, and if you type enter, it opens. If you move slightly your cursor then Software manager is selected back.
- This works as expected in application menus

**Comment by DirkHaar** [comment]
You should more precisely write "system menu", as this behaviour does
not occur in application menus, e.g. Thunderbird or Firefox.

Am 09.01.22 um 10:55 schrieb Loufute:

**Comment by Loufute** [comment]
> You should more precisely write "system menu", as this behaviour does not occur in application menus, e.g. Thunderbird or Firefox. 
Indeed, thank you for the nice suggestion, I updated title and message.

**Comment by Loufute** [comment]
This has been addressed in Linux Mint 21(.1?), thanks guys!
